### ISSUE TITLE
In firefox form-control on a number element cuts off native controls

### ISSUE THREAD

**Issue by andre-d** [twbs/bootstrap#15926]
http://jsbin.com/buhajolumu/

In firefox (36.0, but I have observed this in older firefox) the "up" button gets cut off when form-control is applied to it.  I am not sure if this is a firefox issue or a styling issue.

**Comment by twbs-lmvtfy** [comment]
Hi @andre-d!

You appear to have posted a live example (http://jsbin.com/buhajolumu/edit), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* [W001](https://github.com/twbs/bootlint/wiki/W001): `<head>` is missing UTF-8 charset `<meta>` tag
* [W002](https://github.com/twbs/bootlint/wiki/W002): `<head>` is missing X-UA-Compatible `<meta>` tag that disables old IE compatibility modes
* [W003](https://github.com/twbs/bootlint/wiki/W003): `<head>` is missing viewport `<meta>` tag that enables responsiveness

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by kkirsche** [comment]
I believe this is a duplicate of #15832

**Comment by andre-d** [comment]
Fixed http://jsbin.com/palifawefu/2/

**Comment by andre-d** [comment]
Should I create an upstream issue with mozilla?   #15832 only links to webkit related ones.

**Comment by kkirsche** [comment]
I'll file one with Mozilla as I'm experiencing it to. In a min or two I'll add a link here and will pull request this to the wall of browser bugs

**Comment by andre-d** [comment]
Gah, nevermind, this is a rendering issue in firefox all together regardless of form-control....the top of the button just has no border and appears cut off but the rest has a border *shrugs* Closing

**Comment by andre-d** [comment]
I am not sure if it goes on the wall of bugs when it is unrelated to the styling bootstrap applies to it or not.

**Comment by kkirsche** [comment]
After further research standard forms on OS X in Firefox are not showing caps lock up arrows anyways on OS X.

Wall of bugs as I understand it is usually for upstream issues that can't / won't be fixed from Bootstrap.

Do you have an example of a form where firefox displays an UP arrow when caps lock is enabled?

**Comment by andre-d** [comment]
Caps lock is unrelated to this issue, in some browsers it appears on password forms (Firefox does not do it yet).  The arrows in this example are up down arrows for increasing and decreasing the number in the field.  The bug is that firefox in general cuts off the border on the arrow at the top of a number field (regardless of form-control).

**Comment by kkirsche** [comment]
Not the case for me on Firefox 36.0 @andre-d 

![screen shot 2015-02-25 at 3 55 26 pm](https://cloud.githubusercontent.com/assets/947110/6380318/c41be3ba-bd06-11e4-9808-e8405c9dff2f.png)

**Comment by andre-d** [comment]
This is what it looks like on windows:

![untitled](https://cloud.githubusercontent.com/assets/714162/6380363/178c322a-bd07-11e4-9d61-f4deeab72c29.png)

The top up arrow visually is expected to have a border at the top (like the bottom down arrow has at the bottom and top)

I assume it is a firefox, backend specific rendering issue
